Userland/Libraries/LibWebSocket/WebSocket.cpp

```diff
@@ -504,9 +499,22 @@ void WebSocket::read_frame()
         // We can safely ignore the pong
         return;
     }
-    if (op_code == WebSocket::OpCode::Continuation) {
-        // FIXME: Support fragmented frames
-        TODO();
+
+    if (!is_final_frame) {
+        if (op_code != WebSocket::OpCode::Continuation) {
+            // First fragmented message
+            m_fragmented_data_buffer = ByteBuffer::create_uninitialized(0).release_value_but_fixme_should_propagate_errors();
```

Comment:
Can we skip this step? or maybe call a resize() or similar method? if the fragmentation buffer is shared b/w messages it might be nice to keep the reserved capacity around

Reply:
I have updated the PR, I know there should be a better memory management here, but it's all I can do for now.

Tested with Ladybird on Ubuntu.

https://github.com/SerenityOS/serenity/assets/3373490/a9041a8c-88fc-445b-a912-e82890dd56ef

Reply:
Looks great! Only issue is that you pushed a new commit instead of fixing up the existing one :)

You should do this to resolve it:
```
git fetch origin  # or upstream, or ...
git rebase -i origin/master
# mark your second commit as "f" or "fixup" 
# close the text editor
git push -f
```

Per the contributing guidelines: "Amend your existing commits when adding changes after a review, where relevant."

Reply:
Fixed, thanks